newsfragments/42343.feature.rst

```diff
@@ -0,0 +1 @@
+New function ``create_dataset_aliases`` added to DatasetManager for DatasetAlias creation.
```

Comment:
`DatasetManager` is not part of the public interface right? Do we need to call out new methods in a newsfragment?

Reply:
It is, you can swap it out with `[core] dataset_manager_class`.

Reply:
Thanks for the infos, I did not know that :)